### DISCUSSION TITLE
Add osu!stable multiplayer song beatmap details hover

### DISCUSSION THREAD

**Discussion by nanashi-1** [ppy/osu#24497] [Ideas] [answered]
osu!stable
![Screenshot from 2023-08-11 08-29-29](https://github.com/ppy/osu/assets/54083498/d65b7a3e-a266-4d87-a8af-91b3dc67b37c)

osu!lazer
![Screenshot from 2023-08-10 15-23-48](https://github.com/ppy/osu/assets/54083498/112d5b93-1cb4-4335-a2fa-b0702e7c1004)

**Comment by Joehuu** [comment, marked as answer]
Duplicate of https://github.com/ppy/osu/issues/11306
